Warui-K/django-practice
# [Finishes #164350171]Creates User Registration app

#### What does this PR do?
Creates the user registration app
#### Description of Task to be completed?
- [x] Create a form for registration
- [x] Create a database for user registration
#### How should this be manually tested?
1. Fork and clone the repo
2. Source a virtual environment
```virtualenv venv --no-site-packages```
```source env/bin/activate```
3. ```Pip install requirements.txt```
4. ```python manage.py migrate```
5. ```python manage.py createsuperuser```
6. ```python manage.py runserver```
7. Goto localhost:8000/register
8. Register the new user 
9. Confirm registration on http://127.0.0.1:8000/admin

#### What are the relevant pivotal tracker stories?
#164350171
